app/src/main/res/layout/activity_home.xml

```diff
@@ -18,4 +18,23 @@
         android:overScrollMode="never"
         android:clipToPadding="false"/>
 
+    <android.support.v7.widget.AppCompatImageButton
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:src="@drawable/ic_hamburger"
+        android:layout_centerHorizontal="true"
+        android:layout_marginStart="180dp"
+        android:onClick="hamburgerclicked"
```

Comment:
On click listener will be in Java File not in xml.

Reply:
Done!